cmd/soroban-rpc/lib/preflight/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "preflight"
+version = "0.1.0"
+
+[lib]
+crate-type = ["staticlib"]
+
+[dependencies]
+libc = "0.2.2"
+soroban-env-host = {workspace = true, features = ["vm"]}
+
+[profile.release]
+panic = 'unwind'
```

Comment:
❗ This profile configuration is generating warnings:

```
warning: profiles for the non root package will be ignored, specify profiles at the workspace root:
package:   /Users/leighmcculloch/Code/soroban-tools/cmd/soroban-rpc/lib/preflight/Cargo.toml
workspace: /Users/leighmcculloch/Code/soroban-tools/Cargo.toml
```

The warning is indicating it is ignored because profiles are only read out of the workspace Cargo.toml, not crate Cargo.toml files.

Unfortunately if we need one crate to have a different profile to other crates, then we'll need to create a new profile, such as `release-with-panic-unwind`, and then manually build this crate with that different profile.

Reply:
Thanks, I've done that